src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,39 @@
+package codeu.controller;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+
+/** Servlet class responsible for the admin page. */
+public class AdminServlet extends HttpServlet {
+
+  private static final String[] permittedUser = {"adminTest"};
+
+  private boolean contains(String[] arr, String target){
```

Comment:
Consider using a data structure that has an lookup function or:

https://stackoverflow.com/questions/1128723/how-can-i-test-if-an-array-contains-a-certain-value